src/Test/rockets.test.js

```diff
@@ -0,0 +1,36 @@
+import {
+  render, screen, fireEvent, waitFor,
+} from '@testing-library/react';
+import { Provider } from 'react-redux';
+import store from '../redux/store';
+import Rockets from '../components/Rockets/rockets';
+
+describe('Rockets Component', () => {
+  test('renders rocket names', async () => {
+    render(
+      <Provider store={store}>
+        <Rockets />
+      </Provider>,
+    );
+
+    await waitFor(() => {
+      expect(screen.getByText('Falcon 1')).toBeInTheDocument();
+    });
+  });
+
```

Comment:
- [ ] Please have your [rockets.test.js](https://github.com/IsmaelMastronardi/spacex-group-project/pull/37/files#diff-a565c57615495332a30abfead6ba45b98f457ff0fe2ec656d8ee96dfc4310211) test failing so make sure that you fix that. :pray: 
  
  ![image](https://github.com/IsmaelMastronardi/spacex-group-project/assets/92303891/b0f7763b-7259-4a01-8639-9d14186ce6af)
  ![image](https://github.com/IsmaelMastronardi/spacex-group-project/assets/92303891/a8c5dd94-507c-41eb-a69d-89a4cee06147)

Reply:
Hello, thats weird, because when i hit npm test, the test are passing:
![image](https://github.com/IsmaelMastronardi/spacex-group-project/assets/111823606/c2f868a6-5dee-4b71-9a8c-ba9208e19e32)

![image](https://github.com/IsmaelMastronardi/spacex-group-project/assets/111823606/7cd78053-4112-47bf-bd0c-00a23e1eb2b9)